src/main/resources/data-repository-openapi.yaml

```diff
@@ -343,7 +452,7 @@ definitions:
       - to
     properties:
       name:
-        $ref: '#/definitions/IdentifierNameModel'
+        $ref: '#/definitions/ObjectName'
```

Comment:
This comment applies to the RelationshipModel description: if the documentation about the cardinality still applies, it should be moved to the RelationshipTermModel object

Reply:
Let's discuss. I think this okay as is.